google-cloud-debugger/docs/instrumentation.md

```diff
@@ -50,6 +50,17 @@ are finished being evaluated. Be aware the more breakpoints are created,
 or the harder to reach the breakpoints, the more resource the debugger
 agent would need to consume.
 
+### Configuration
+
+The default configuration enables Stackdriver instrumentation features to run on
+Google Cloud Platform. You can easily configure the instrumentation library if 
+you want to run on a non Google Cloud environment or you want to customize 
+the default behavior.
+
+See the
+[Configuration Guide](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/stackdriverguides/instrumentation_configuration)
```

Comment:
Is this part of the path correct? `/stackdriverguides/`
It seems like it should be `/stackdriver/guides/` instead. 
This occurs in a number of locations.